### PR TITLE
created ontime.js

### PR DESCRIPTION
**Pull request by aakashrajput**
- i tested the code it is working fine..
  the thing i did makin global.Ontime = {}; in app.js
  so it is good to go with the other codes :D

### CODE REVIEW

File: users.js [hunk 1]
```diff
@@ -627,9 +627,10 @@ class User {
 		} else {
 			this.send('|nametaken|' + name + "|Your authentication token was invalid.");
 		}
-
+               	Ontime[userid] = Date.now();
```
Comment (panpawn) [inline users.js:630]: spacing

File: app.js [hunk 2]
```diff
@@ -117,6 +117,8 @@ global.Ladders = require(Config.remoteladder ? './ladders-remote.js' : './ladder
 
 global.Users = require('./users.js');
 
+global.Ontime = {};
```
Comment (panpawn) [inline app.js:120]: we would need to add this global to .eslintrc too

File: chat-plugins/ontime.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+		if (isConnected) {
+			this.sendReplyBox('<b><font color="' + Wisp.hashColor(userid) + '">' + userid + '</font></b>\'s total ontime is <b>' + displayTime(convertTime(totalOntime)) + '</b>.' + ' Current ontime: <b>' + displayTime(convertTime((currentOntime))) + '</b>.');
+		} else {
+			this.sendReplyBox('<b><font color="' + Wisp.hashColor(userid) + '">' + userid + '</font></b>\'s total ontime is <b>' + displayTime(convertTime(totalOntime)) + '</b>.' + ' Currently not online.');
+		}
+	},
+	ontimehelp: ["/ontime - Shows how long in total the user has been on the server."],
+
+
```
Comment (panpawn) [inline chat-plugins/ontime.js:51]: why two spaces here

File: app.js (hunk 2 above)
Comment (panpawn) [inline app.js:120]: but im against this even being a global

File: chat-plugins/ontime.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
+'use strict';
+/********************
+ * OnTime Commands
+ * This file contains commands that keep track of users activity, and for how long user is online.
+********************/
+let moment = require('moment');
+
+
+
```
Comment (panpawn) [inline chat-plugins/ontime.js:9]: too many spaces

**Comment by panpawn** [comment]
this isn't how we would do ontime

**Comment by aakashrajput** [comment]
Hmm

**Pull request closed without merging**